scripts/build-glean-metadata.py

```diff
@@ -139,7 +139,9 @@ def etl_snake_case(line: str) -> str:
                 app_metrics[metric.identifier] = dict(
                     metric.definition,
                     name=metric.identifier,
-                    annotation=annotations_index.get(app_name, {}).get(metric.identifier),
+                    annotation=annotations_index.get(app_name, {})
+                    .get("metrics", {})
```

Comment:
This is black's behavior on long chainable arguments? Imo not the prettiest look (just throwing my opinion out there).

Reply:
Agreed, it's not great. Maybe we can find a better way of expressing this in the future.

Reply:
Maybe wrapping the whole expression in another set of parenthesis might add an indent. Nothing worth nitpicking over though.

Reply:
Yeah, actually this helped a bunch! I was in the file anyway to revert the change for the annotations URL so it was no trouble.